authors/apps/articles/models.py

```diff
@@ -0,0 +1,59 @@
+from django.db import models
+from django.utils.text import slugify
+from django.db.models.signals import pre_save
+from django.dispatch import receiver
+from authors.apps.core.utils import random_string_generator
+from authors.apps.core.models import TimeModel
+from authors.apps.profiles.models import Profile
+
+class Article(TimeModel):
```

Comment:
Since you have new models and migrations you will need to add instructions of  how the rest can apply these changes to their own DBs. I have encountered and error when trying to run the migrations